pages/Hercules'-Pickaxe.md

```diff
@@ -0,0 +1,3 @@
+Hercules' Pickaxe is an item in the [tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tools) category.
+
+This tool is similar in function to an [Ore Crusher](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Crusher) or an [Electric Ore Grinder](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ore-Grinder): it crushes iron ore mined with it into [iron dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Iron-Dust), and gold ore into [gold dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Dust).
```

Comment:
You could add a header here.

And doesn't it also double other ore drops?
Or am I mxing things up here?

Reply:
It absolutely does, I just forgot to mention it